src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -606,8 +606,18 @@ function (IndexedServiceContainer $c) {
             } catch (\InvalidArgumentException $e) {
                 throw new \RuntimeException(sprintf('Formatter not recognised: "%s"', $formatterName));
             }
+
             $c->set('event_dispatcher.listeners.formatter', $formatter, ['event_dispatcher.listeners']);
         });
+
+        if (getenv('GITHUB_ACTIONS')) {
```

Comment:
The standard place to read environment variables from is actually `$_SERVER`. `getenv` and `putenv` are not threadsafe.

Reply:
Surely we're safe on a single-threaded CLI app?

Reply:
I'm quite sure some crazy devs have created a web runner for phpspec...

Reply:
Even then, surely `GITHUB_ACTIONS` is set to the same value on all threads? ;)